### DISCUSSION TITLE
GraphQL query does not return result

### DISCUSSION THREAD

**Discussion by phandc** [grafana/grafana-infinity-datasource#128] [Specific APIs]
Hi, I'm currently trying to figure out the Infinity datasource plugin. 
I have a localhost server and I try to get the data through GraphQL query from Grafana. Here is my example:

![Screenshot from 2021-08-17 23-51-49](https://user-images.githubusercontent.com/42259411/129767987-f7cdc3d4-71ca-4ffb-83ab-0297f79a6174.png)

My localhost server returns the response 200. When I use Query Inspector to discover the query, it still keeps loading. Therefore, it returns nothing.

![Screenshot from 2021-08-17 23-45-00](https://user-images.githubusercontent.com/42259411/129767994-7afe7771-2713-4dbb-9681-c339cf3249ee.png)

How can I do to get the result from the query? 
Thanks for your time!

**Comment by yesoreyeram** [comment]
Can you inspect the `Data` tab instead of `Query`. Also chrome network tab results may help to identify the issue. ( chrome+ F12+ network tab ).

**Reply by phandc** [reply]
@yesoreyeram thanks for your reply,
Here are the images that you mentionend above:

![Screenshot from 2021-08-18 08-50-28](https://user-images.githubusercontent.com/42259411/129824192-646fe1aa-80f0-4325-baf8-b64b5182a4ca.png)

![Screenshot from 2021-08-18 08-53-40](https://user-images.githubusercontent.com/42259411/129824195-3886ab12-bbd9-4ce7-8154-daa383d6929a.png)

**Reply by yesoreyeram** [reply]
Ok from the network it seems you are getting response. So I think you need a proper root value set in your query. If you see the response tab, that will give you some idea.
